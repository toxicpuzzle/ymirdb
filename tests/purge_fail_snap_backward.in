SET c 7
SET b 3 5 c
SET a 1 2 b c
SET d a b c
SNAPSHOT
DEL a
LIST ENTRIES
DEL a
CHECKOUT 1
LIST ENTRIES
SNAPSHOT
SET b 1 2 3
DEL d
DEL a
PURGE b
LIST ENTRIES
PURGE a
LIST ENTRIES
PURGE b
PURGE c
PURGE d
LIST ENTRIES
bye