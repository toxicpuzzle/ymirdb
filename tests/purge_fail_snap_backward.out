> ok

> ok

> ok

> ok

> saved as snapshot 1

> not permitted

> d [a b c]
a [1 2 b c]
b [3 5 c]
c [7]

> not permitted

> ok

> d [a b c]
a [1 2 b c]
b [3 5 c]
c [7]

> saved as snapshot 2

> ok

> ok

> ok

> not permitted

> b [1 2 3]
c [7]

> not permitted

> b [1 2 3]
c [7]

> not permitted

> not permitted

> ok

> b [1 2 3]
c [7]

> bye